targets/targets.json

```diff
@@ -8778,6 +8777,65 @@
         "mbed_ram_start"        : "0x20000000",
         "mbed_ram_size"         : "0x8000"
     },
+    "NU_PFM_M2351_NS": {
+        "inherits": ["NSPE_Target", "NU_PFM_M2351_CM"],
```

Comment:
`NU_PFM_M2351_CM`? is it like that there is common between secure- and non-secure-side? If it is so why it's not named as `NU_PFM_M2351`?

Reply:
`NU_PFM_M2351` is reserved for special target like **No TrustZone** target, though it is not supported currently.